### PR TITLE
Updating and Relocating Repository Structure and Maintainers Instructions

### PR DESCRIPTION
**Pull request by VRamakrishna**
- Replaced Hyperledger references with LFDT references.
- Moved three files (`repository-structure.md`, `MAINTAINERS-guidelines.md`, and `SAMPLE-MAINTAINERS.md`) from `guidelines` to `governing-documents` folder.
- Fixed typos and outdated links wherever I could find any.

### CODE REVIEW

**Comment by VRamakrishna** [comment]
_Question_: given the relocation of `MAINTAINERS-guidelines.md`, should I rename the file as well? How about `MAINTAINERS-file.md`?

_Answer_: Renaming to `MAINTAINERS-file.md` as agreed during the Sept 12 2024 TOC meeting.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (VRamakrishna) [inline governing-documents/MAINTAINERS-guidelines.md]: _Question_: should this be the LFDT TAC or the project's TSC? I can see this being a concern for both bodies.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (VRamakrishna) [inline governing-documents/MAINTAINERS-guidelines.md]: Same question as above: LFDT TAC or the project's TSC?

File: governing-documents/SAMPLE-MAINTAINERS.md
Comment (VRamakrishna) [inline governing-documents/SAMPLE-MAINTAINERS.md]: Same question as above: LFDT TAC or the project's TSC?

File: governing-documents/MAINTAINERS-guidelines.md
Comment (VRamakrishna) [inline governing-documents/MAINTAINERS-guidelines.md]: I forgot to update this, but I'll do so after my above question is answered.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (petermetz) [reply governing-documents/MAINTAINERS-guidelines.md]: @VRamakrishna I'd be fine with saying both. It's probably the TSC's responsibility to add the maintainers.md file (if we assume that TSC is mostly the same set of people as the project maintainers) but in the cases when it isn't it makes sense for the TSC to have this responsibility in addition to the TAC also paying attention to it.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (petermetz) [reply governing-documents/MAINTAINERS-guidelines.md]: I'd say TSC for this one. The guiding principle being that we are trying to push as much to the edge as possible (e.g. if the TSC can be responsible for something then let's not make it the responsibility of the TAC)

File: governing-documents/SAMPLE-MAINTAINERS.md
Comment (petermetz) [reply governing-documents/SAMPLE-MAINTAINERS.md]: Here I'd probably say TSC, but I would also be partial to mentioning both of them as an either or kinda thing.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (petermetz) [reply governing-documents/MAINTAINERS-guidelines.md]: @VRamakrishna I put down my thoughts, these are just a vote though. No strong opinions on either of those answers that I provided and if someone else from the TOC feels strongly in the opposite direction I'm happy to go along.

**Review by petermetz — APPROVED** [review]
LGTM in general assuming that the comments/questions are all cleared up before merge.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (swcurran) [reply governing-documents/MAINTAINERS-guidelines.md]: How about — the project TSC **MUST** monitor that all repositories have a maintainers.md file, and the TAC **SHOULD** send out periodioc notifications about missing files.

File: governing-documents/SAMPLE-MAINTAINERS.md
Comment (VRamakrishna) [reply governing-documents/SAMPLE-MAINTAINERS.md]: Personally, I think the project's TSC is enough (following the push-to-the-edge principle you mentioned above.)

File: governing-documents/MAINTAINERS-guidelines.md
Comment (VRamakrishna) [reply governing-documents/MAINTAINERS-guidelines.md]: I think this ought to be the project's TSC too.

File: governing-documents/MAINTAINERS-guidelines.md
Comment (VRamakrishna) [reply governing-documents/MAINTAINERS-guidelines.md]: I'm agnostic on this myself. I can say "TAC or TSC".

Comment (VRamakrishna) [reply governing-documents/MAINTAINERS-guidelines.md]: Accepting Stephen's suggestion.

**Comment by VRamakrishna** [comment]
@tkuhrt @arsulegai I've made the changes as discussed and agreed on in today's TOC meeting. Thanks for your questions and feedback. Please review and let me know if you find any mistakes. (I can push updates if required until tomorrow, after which I will be unavailable for 2 weeks.)

**Comment by tkuhrt** [comment]
Changes were added via https://github.com/LF-Decentralized-Trust/governance/pull/23

**Pull request closed without merging**
